script/Ciel/Day 10/10_01_CIEL10_5.txt

```diff
@@ -441,92 +495,102 @@ I said that vampires were frightening.
 [sha:48e31f72d6c7052929b118e5c7c14f7b18483d81]{
 -- Page 316, Offset 27924.
 -- 戦いは、殺し合いは恐ろしいと。
-That killing or being killed in battle was frightening.
+That I'm terrified of killing, or being killed in battle.
 }
 [sha:07cbc5a2bb15c84bf972d712a7203482e8b365c1]{
 -- Page 316, Offset 27925.
 -- それらは瞬間的なものにすぎない。
-But those are all transient fears, nothing more.
+But those are all fears that come and go.
 }
 [sha:22c4cf2cb35155a5930f118b58dc0df8328a3bde]{
 -- Page 316, Offset 27926.
 -- 俺にとっては、この毎日こそが常にある恐怖だった。
-To me, the constant fear lies in each and every single day.
+To me, each passing day represents a fear that will never fade.
```

Comment:
Don't love the noun swap, since as written we have a nice progression of days -> fear, fear -> explanation. If we start this one on fear (general) and then the next with fear (specific), it feels like more of a jump. 
I think this phrasing also loses a bit of the tie that it is the days themselves, and their passing, that instills fear. Without the jp to back it, it reads more to me like there is some _other_ fear that is present every day.

Reply:
The case against the noun switch I do sort of understand, though I don't think its particularly unclear either way. Honestly 'passing' in and of itself might be unnecessary considering the next line. Or rather, it undermines the next line. In which case you'd rather have it be more focused on his mundane life.

God this line is annoying, but it's sort of got a lot of factors to it. Not only the ones that come after, but also how it contrasts with some of the earlier stuff.

So something like:
>To me, it is these mundane days that entail a constant fear.

transient <> constant
no passage to lead into the next line
mundanity to describe his current situation as opposed to the fear represented by vampires.
'it is' to give further contrast
I think entail is just a bit more elegant than represent, and it comes across a bit stronger.



For reference to relevant lines:
>But those are all transient fears, nothing more.
...
>To me, it is these mundane days that entail a constant fear.
>The fear that my precious time on earth won't last until tomorrow.
...
>I have to suppress the part of me that fears the mundane passing of each day.

This last one has to be adjusted depending on this line, but it can include 'passing', since it's the final round out of what's being told.